app/utils/JsonHelper.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import org.json.XML
+import play.api.Logger
+import play.api.libs.json.JsObject
+import play.api.libs.json.Json
+
+object JsonHelper {
+
+  def convertXmlToJson(xml: String): JsObject =
+    try {
+      Json.parse(XML.toJSONObject(xml).toString).as[JsObject]
+    } catch {
+      case error: Exception =>
+        Logger.error(s"Failed to convert xml to json with error: ${error.getMessage}")
```

Comment:
What do these error messages look like? We don't want the possibility of the error messages containing any content from the body of the xml